### PR TITLE
issue: 801695 VMA OOB sockperf tuning for low latency

### PR DESCRIPTION
**Pull request by rosenbaumalex**
provide best tuning for sockperf ping-pong latency test using: VMA_SPEC=10

Signed-off-by: Alex Rosenbaum <alexr@mellanox.com>

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/accl-libvma-pr/243/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/accl-libvma-pr/258/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/accl-libvma-pr/260/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/accl-libvma-pr/292/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/accl-libvma-pr/311/ for details (Mellanox internal link).

**Comment by rosenbaumalex** [comment]
bot:retest

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/accl-libvma-pr/352/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/accl-libvma-pr/334/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/accl-libvma-pr/341/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/accl-libvma-pr/359/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/accl-libvma-pr/357/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/accl-libvma-pr/375/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/accl-libvma-pr/388/ for details (Mellanox internal link).

**Comment by OphirMunk** [comment]
@rosenbaumalex PR needs rebasing

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/accl-libvma-pr/370/ for details (Mellanox internal link).

File: src/vma/main.cpp [hunk 1]
```diff
@@ -410,6 +410,9 @@ void print_vma_global_settings()
 	vlog_printf(log_level,"---------------------------------------------------------------------------\n");
 
 	switch (safe_mce_sys().mce_spec) {
+	case MCE_SPEC_SOCKPERF_LL_10:
+		vlog_printf(VLOG_INFO, " sockperf ping-pong low latency profile spec\n");
```
Comment (OphirMunk) [inline src/vma/main.cpp]: Keep upper letters
```
vlog_printf(VLOG_INFO, " sockperf ping-pong Low Latency Profile Spec\n");
```

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/accl-libvma-pr/393/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/accl-libvma-pr/375/ for details (Mellanox internal link).

**Pull request merged**